### DISCUSSION TITLE
Raspishake

### DISCUSSION THREAD

**Discussion by heiler** [xspanger3770/GlobalQuake#193] [Ideas]
Is there any chance to see the data from Raspishake?
https://data.raspberryshake.org/fdsnws/station/1/

**Comment by YacineBoussoufa** [comment]
If you have a RaspberryShake you can connect to it via your local Seedlink server.

**Comment by AyanamiJoss** [comment]
The official Raspberry Shake Seedlink server is **not provided** by the GlobalQuake programming team, which is why it is not added to the program. **You can purchase access to Raspberry Shake officially**: https://raspberryshake.org/data-center/
If you have a Raspberry Shake running officially, you can access it through its configuration link, either http://rs.local/ or http://192.168.x.x/ (the "x" are variants, so it will depend on the things you have running locally and in the background)
